src/api/strings/shared/random.lua

```diff
@@ -4,9 +4,7 @@
 ---@public
 function API_Strings:randomString(length)
     local charset <const> = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
-    length = length or 10
-    local randomString = ""
-    for i = 1, length do
+    for i = 1, (length or 10) do
```

Comment:
Sorry i removed randomString variable
```suggestion
    local randomString = ""
    for i = 1, (length or 10) do
```